src/utils/properties.ts

```diff
@@ -26,3 +26,7 @@ export const CACHE_SERVER = getEnvironmentVariable("CACHE_SERVER");
 export const SHOW_SERVICE_OFFLINE_PAGE = getEnvironmentVariable("SHOW_SERVICE_OFFLINE_PAGE");
 
 export const CHS_API_KEY = getEnvironmentVariable("CHS_API_KEY");
+
+export const INTERNAL_API_URL = getEnvironmentVariable("INTERNAL_API_URL");
+
+export const PRIVATE_SDK_FEATURE_FLAG = getEnvironmentVariable("PRIVATE_SDK_FEATURE_FLAG");
```

Comment:
Steve wanted us to follow the CH style guide for feature flag naming - https://github.com/companieshouse/styleguides/blob/master/feature_flag.md